fuzzers/000-init-db/Makefile

```diff
@@ -82,15 +84,18 @@ status:
 $(DB_FILES_PATH):
 	touch $@
 
-pushdb: $(DB_FILES_PATH)
+${XRAY_FAMILY_DIR}/${XRAY_PART}:
+	mkdir -p ${XRAY_FAMILY_DIR}/${XRAY_PART}
+
+pushdb: $(DB_FILES_PATH) ${XRAY_FAMILY_DIR}/${XRAY_PART}
 	@true
 
 run:
-	$(MAKE) clean
+	rm -rf $(BUILD_DIR) run.${XRAY_PART}.ok
```

Comment:
Should there be a
```makefile

clean-${XRAY_PART}:
	rm -rf $(BUILD_DIR) run.${XRAY_PART}.ok

run:
	$(MAKE) clean-${XRAY_PART}
```

Reply:
I'd argue no.  If you are running interactively, `make clean` does the right thing. If you are doing multiple parts, you won't typically be looking at one fuzzer.